### DISCUSSION TITLE
[破事氵]听说Knock Knock和火遍新艾利都的游戏原神合作了？

### DISCUSSION THREAD

**Discussion by Erdbronie** [share121/inter-knot#3462] [其他]
![屏幕截图 2024-08-22 144245](https://github.com/user-attachments/assets/25bf9f56-892d-414a-a65f-bc6b686d999a)
![屏幕截图 2024-08-22 144304](https://github.com/user-attachments/assets/b5d13f9a-8bc6-4597-8a86-a24f3a183ed4)

**Comment by hk0852** [comment]
没啥用，现在可以暂时岁月安好是因为米哈游还没有正式接触到社交软件（米游社搞没搞好，并且也不是视频平台也不包括平时的日常交流比如通讯仅仅只是帖子，视频日常交流比如通讯和帖子的差别影响力就不用我说了吧），到了那一刻，环境会瞬间变坏，当年的qq群聊被爆破网暴的现象就会重演，想要破局就要搞定虚拟技术并且利用虚拟技术的优势跟国家的通讯运营商展开合作，这样才能绕开腾讯和其他手机厂商和应用厂商，因为社交软件没有人可以比得过腾讯，就连他自己本身都不行，比如微信和qq，对于所有人来讲qq可以舍弃因为只是占有了游戏和娱乐，但是微信确确实实地占领了国内（包括部分外国人）的每一台手机以及钱包银行卡，出行地图熟人交流，饮食，包括娱乐只要你还需要接触社会（除了部分特殊工作）就完全离不开

**Comment by hk0852** [comment]
因为米哈游还没有准备好从游戏公司或者靠ip影响力吃饭的公司转变到互联网公司
